---
title: ""
output:
  html_document:
    df_print: kable
    toc: true
    toc_float : yes
    toc_depth : 4
---

```{r fxns, include = FALSE}
library('htmlTable')
make_list <- function(items) {
  paste0("<ul>", sprintf('<li>%s</li>', items), '</ul>', collapse = '')
}
make_href <- function(url, text){
  paste0("<div style='text-align:center; font-size:large; font-weight:bold'><a href=", url ,">", text, "</a></div>")
}
make_href2 <- function(url, text){
  paste0("<a href=", url ,">", text, "</a>")
}
process_entry <- function(dat) {
  if (grepl(pattern = "https://satijalab.org/img/vignette_images", x = dat$image)) {
    img <- paste0('![](', dat$image, '){width=3000px}')
  } else if (grepl(pattern = "assets/", x= dat$image)) {
    img <- paste0('![](', dat$image, '){width=3000px}') 
  } else {
    img <- paste0('![](', 'image/', dat$image, '){width=3000px}')
  }
  link <- dat$link
  if(!is.null(link)){
    go.button <- paste0('<a class="btn btn-primary" href="', link, '" role="button" style="width:100%" target = "_blank">GO</a>')
  } else {
    go.button <- paste0('<a class="btn btn-primary" href="', link, '" role="button" style="width:100%" target = "_blank" disabled>GO</a>')
  }
  data.frame(
    title = make_href(url = link, text = dat$title),
    img = img,
    desc = dat$summary,
    btn = go.button
  )
}
process_wrapper_entry <- function(dat) {
  data.frame(
    Package = dat$name,
    Vignette = make_href2(url = dat$link, text = dat$title),
    Reference = make_href2(url = dat$reference, text = dat$citation),
    Source = make_href2(url = dat$source, text = dat$source)
  )
}
make_vignette_card_section <- function(vdat, cat) {
  vignettes <- vdat[[cat]]$vignettes
  dat <- data.frame(title = character(), img = character(), desc = character())
  for (v in 1:length(x = vignettes)) {
    dat <- rbind(dat, process_entry(vignettes[[v]]))
    if(nrow(x = dat) == 3 | v == length(x = vignettes)){
      colnames(dat) <- NULL
      dat <- t(dat)
      if (ncol(x = dat) == 2) {
        print(htmlTable(
          dat, 
          align = '|l|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 50%", 
          css.class = "two-column-htmltable"
        ))
      } else if (ncol(x = dat) == 1){
        print(htmlTable(
          dat, 
          align = '|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 100%", 
          css.class = "one-column-htmltable"
        ))
      } else {
        print(htmlTable(
          dat, 
          align = '|l|l|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 30%"
        ))
      }
      dat <- data.frame(title = character(), img = character(), desc = character())
    }
  }
}
```

```{r yaml, include = FALSE}
library(yaml)
vdat <- read_yaml(file = "research.yml")
```

```{=html}
<style>
.two-column-htmltable {
  width: 66.666%;
}
.one-column-htmltable {
  width: 33.333%;
}
</style>
```

<!-- Although I am currently interested in computational omics data analysis and building bioinformatics analysis platforms, I have also done some research on statistical learning and statistical software development for some time. -->

<br>

# Omics Data Analysis

<!-- <p style="text-align: justify"> In high throughput sequencing era, genetic information is complex, chaotic and often incomprehensible. To this end, Bioinformatics has evolved to analyze and summarize massive sequencing data sets, and data scientific methods have been developed to investigate biological systems and cellular dynamics by using these large volumes of processed omics data. </p> -->

<p style="text-align: justify"> 
I have designed software platforms and tools to process and analyze large sets of sequencing data in order to find similarities across diverse omics datasets and to build network models to interrogate databases against each other to help biological scientists to build hypothesis. 
</p>

<p style="text-align: justify"> Currently, we are developing additional software tools to analyze spatial transcriptomics datasets and to infer spatial information of cellular systems. </p>

<br>

```{r results='asis', echo=FALSE, warning=FALSE, message = FALSE}
make_vignette_card_section(vdat = vdat, cat = 1)
```

<br>

# Statistical Learning/Computing

<p style="text-align: justify"> 
Computing is an essential part of statistical analysis where building appealing models of data as well as visualizing such data is crucial. A large portion of my work before I moved into computational biology which included building and designing data structures for capturing essential information from the data to either summarize or visualize complex information. 
</p>

<p style="text-align: justify"> 
We have built graph models to capture essential information from data for the purpose of statistical learning, and we have also designed software tools to visualize and help analyzing such data (i.e. Dandelion Plot). 
</p>

<br>

```{r results='asis', echo=FALSE, warning=FALSE, message = FALSE}
make_vignette_card_section(vdat = vdat, cat = 2)
```