---
title: ""
output:
  html_document:
    df_print: kable
    toc: true
    toc_float : yes
    toc_depth : 4
---

```{r fxns, include = FALSE}
library('htmlTable')
make_list <- function(items) {
  paste0("<ul>", sprintf('<li>%s</li>', items), '</ul>', collapse = '')
}
make_href <- function(url, text){
  paste0("<div style='text-align:center; font-size:large; font-weight:bold'><a href=", url ,">", text, "</a></div>")
}
make_href2 <- function(url, text){
  paste0("<a href=", url ,">", text, "</a>")
}
process_entry <- function(dat) {
  if (grepl(pattern = "https://satijalab.org/img/vignette_images", x = dat$image)) {
    img <- paste0('![](', dat$image, '){width=3000px}')
  } else if (grepl(pattern = "assets/", x= dat$image)) {
    img <- paste0('![](', dat$image, '){width=3000px}') 
  } else {
    img <- paste0('![](', 'image/', dat$image, '){width=3000px}')
  }
  link <- dat$link
  if(!is.null(link)){
    go.button <- paste0('<a class="btn btn-primary" href="', link, '" role="button" style="width:100%" target = "_blank">GO</a>')
  } else {
    go.button <- paste0('<a class="btn btn-primary" href="', link, '" role="button" style="width:100%" target = "_blank" disabled>GO</a>')
  }
  data.frame(
    title = make_href(url = link, text = dat$title),
    img = img,
    desc = dat$summary,
    btn = go.button
  )
}
process_wrapper_entry <- function(dat) {
  data.frame(
    Package = dat$name,
    Vignette = make_href2(url = dat$link, text = dat$title),
    Reference = make_href2(url = dat$reference, text = dat$citation),
    Source = make_href2(url = dat$source, text = dat$source)
  )
}
make_vignette_card_section <- function(vdat, cat) {
  vignettes <- vdat[[cat]]$vignettes
  dat <- data.frame(title = character(), img = character(), desc = character())
  for (v in 1:length(x = vignettes)) {
    dat <- rbind(dat, process_entry(vignettes[[v]]))
    if(nrow(x = dat) == 3 | v == length(x = vignettes)){
      colnames(dat) <- NULL
      dat <- t(dat)
      if (ncol(x = dat) == 2) {
        print(htmlTable(
          dat, 
          align = '|l|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 50%", 
          css.class = "two-column-htmltable"
        ))
      } else if (ncol(x = dat) == 1){
        print(htmlTable(
          dat, 
          align = '|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 100%", 
          css.class = "one-column-htmltable"
        ))
      } else {
        print(htmlTable(
          dat, 
          align = '|l|l|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 30%"
        ))
      }
      dat <- data.frame(title = character(), img = character(), desc = character())
    }
  }
}
```

```{r yaml, include = FALSE}
library(yaml)
vdat <- read_yaml(file = "research.yml")
```

```{=html}
<style>
.two-column-htmltable {
  width: 66.666%;
}
.one-column-htmltable {
  width: 33.333%;
}
</style>
```


<br>

# Omics Data Analysis

I have applied a range of statistical methods and developed software platforms and analytical tools to process and analyze
large-scale sequencing datasets. These tools facilitate the discovery of patterns and similarities across diverse omics datasets,
enabling the construction of statistical models that support robust hypothesis generation.

In various <a href="projects.html">projects</a>, I have integrated data from multiple sources, including:

* <strong>Gene expression </strong>data (RNA-seq, scRNA-seq),
* <strong>DNA methylation </strong>profiles (Bisulfite-seq, RRBS, methylation arrays),
* <strong>Open chromatin</strong> regions (ATAC-seq),
* <strong>Transcription factor binding sites</strong> (ChIP-seq),
* Data from <strong>specialized methods</strong>, such as DRIP-seq and RDIP-seq, to detect DNA-RNA hybrids.
* Information on therapies. biomarkers and drugs from databases of <strong>clinical trials</strong>


# Statistical Analysis

Examples consist of:

* <strong>Multi-task Regression Methods</strong>: These methods are used to simultaneously model multiple related outputs,
enabling efficient handling of data from various sources. In the context of omics data, multi-task
regression can capture shared patterns across related datasets, improving predictive accuracy and
providing insights into interconnected biological processes.

* <strong>Classification Methods</strong>: I employ a variety of classical classification techniques,
such as logistic regression, decision trees, and support vector machines (SVMs),
to categorize data based on features derived from omics datasets.
These methods are essential for distinguishing between different biological states,
such as healthy vs. diseased samples or cell type classifications. By applying these well-established algorithms,
I can create predictive models that provide insights into underlying biological processes and
help identify potential biomarkers. Additionally, I incorporated positive-unlabeled learning when
working with partially labeled data, allowing for the effective classification of samples in
scenarios where only a subset has known annotations.

* <strong>Unsupervised Methods</strong>, e.g. Autoencoders: Autoencoders are neural network models that learn compact,
informative representations of high-dimensional data. In omics analysis, I leverage autoencoders to uncover
hidden structures and reduce data dimensionality, facilitating the discovery of patterns in transcription
factor activity and other biological features. These learned embeddings can further be used to cluster data,
identify relationships, and detect anomalies.






