---
title: ""
output:
  html_document:
    df_print: kable
    toc: true
    toc_float : yes
    toc_depth : 4
---

```{r fxns, include = FALSE}
library('htmlTable')
make_list <- function(items) {
  paste0("<ul>", sprintf('<li>%s</li>', items), '</ul>', collapse = '')
}
make_href <- function(url, text){
  paste0("<div style='text-align:center; font-size:large; font-weight:bold'><a href=", url ,">", text, "</a></div>")
}
make_href2 <- function(url, text){
  paste0("<a href=", url ,">", text, "</a>")
}
process_entry <- function(dat) {
  if (grepl(pattern = "https://satijalab.org/img/vignette_images", x = dat$image)) {
    img <- paste0('![](', dat$image, '){width=3000px}')
  } else if (grepl(pattern = "assets/", x= dat$image)) {
    img <- paste0('![](', dat$image, '){width=3000px}') 
  } else {
    img <- paste0('![](', 'image/', dat$image, '){width=3000px}')
  }
  link <- dat$link
  if(!is.null(link)){
    go.button <- paste0('<a class="btn btn-primary" href="', link, '" role="button" style="width:100%" target = "_blank">GO</a>')
  } else {
    go.button <- paste0('<a class="btn btn-primary" href="', link, '" role="button" style="width:100%" target = "_blank" disabled>GO</a>')
  }
  data.frame(
    title = make_href(url = link, text = dat$title),
    img = img,
    desc = dat$summary,
    btn = go.button
  )
}
process_wrapper_entry <- function(dat) {
  data.frame(
    Package = dat$name,
    Vignette = make_href2(url = dat$link, text = dat$title),
    Reference = make_href2(url = dat$reference, text = dat$citation),
    Source = make_href2(url = dat$source, text = dat$source)
  )
}
make_vignette_card_section <- function(vdat, cat) {
  vignettes <- vdat[[cat]]$vignettes
  dat <- data.frame(title = character(), img = character(), desc = character())
  for (v in 1:length(x = vignettes)) {
    dat <- rbind(dat, process_entry(vignettes[[v]]))
    if(nrow(x = dat) == 3 | v == length(x = vignettes)){
      colnames(dat) <- NULL
      dat <- t(dat)
      if (ncol(x = dat) == 2) {
        print(htmlTable(
          dat, 
          align = '|l|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 50%", 
          css.class = "two-column-htmltable"
        ))
      } else if (ncol(x = dat) == 1){
        print(htmlTable(
          dat, 
          align = '|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 100%", 
          css.class = "one-column-htmltable"
        ))
      } else {
        print(htmlTable(
          dat, 
          align = '|l|l|l|', 
          css.cell = "padding-left: .75em; padding-right: .50em; padding-bottom: .20em; width: 30%"
        ))
      }
      dat <- data.frame(title = character(), img = character(), desc = character())
    }
  }
}
```

```{r yaml, include = FALSE}
library(yaml)
vdat <- read_yaml(file = "research.yml")
```

```{=html}
<style>
.two-column-htmltable {
  width: 66.666%;
}
.one-column-htmltable {
  width: 33.333%;
}
</style>
```


<br>

# Omics Data Analysis

I specialize in leveraging advanced statistical methods and developing cutting-edge software tools to process and analyze large-scale sequencing datasets.
These innovations enable the discovery of meaningful patterns and relationships across diverse omics datasets,
paving the way for robust hypothesis generation and data-driven insights.

In my <a href="projects.html">projects</a>, I integrated multi-omics data from a variety of sources, including:

* <strong>Gene expression </strong>data (RNA-seq, scRNA-seq),
* <strong>DNA methylation </strong>profiles (Bisulfite-seq, RRBS, methylation arrays),
* <strong>Open chromatin</strong> regions (ATAC-seq),
* <strong>Transcription factor binding sites</strong> (ChIP-seq),
* Data from <strong>specialized protocols</strong> and methods, such as DRIP-seq and RDIP-seq, for detecting DNA-RNA hybrids
*  Information on <strong>therapies</strong>, <strong>drugs</strong>, and <strong>biomarkers</strong> from internal and external clinical trial databases

By combining these diverse data types, I created comprehensive models that contribute to understanding complex biological systems.

# Statistical Analysis

I employed a wide range of statistical techniques to extract meaningful insights from complex datasets, including:

* <strong>Survival Analysis</strong>: Utilizing methods such as the Kaplan-Meier Estimator and Cox Proportional Hazards Model to assess time-to-event data and evaluate prognostic factors.
* <strong>Regression Analysis</strong>: Applying linear regression and other predictive modeling techniques to uncover relationships between variables.
* <strong>Classification Methods</strong>: Leveraging algorithms such as logistic regression, elastic net, random forests, support vector machines (SVMs), and positive-unlabeled (PU) learning for accurate predictions and classifications.
* <strong>Unsupervised Methods</strong>: Employing dimensionality reduction and clustering techniques, including hierarchical clustering, PCA, MOFA, and autoencoders, to uncover hidden structures in data.







